File: AspNetMvcUnleashed/0672329980_source/Chapter08/CS/MvcApplication1/MvcApplication1.Tests/Models/FakeMovieRepository.cs
using System.Collections.Generic;
using MvcApplication1.Models;

namespace MvcApplication1.Tests.Models
{
    public class FakeMovieRepository : IMovieRepository
    {
        #region IMovieRepository Members

        public IEnumerable<Movie> ListMovies()
        {
            return null;
        }

        public void CreateMovie(Movie movieToCreate)
        {
        }

        #endregion
    }
}

File: AspNetMvcUnleashed/0672329980_source/Chapter08/VB/MvcApplication1/MvcApplication1.Tests/Models/FakeMovieRepository.vb
Public Class FakeMovieRepository
    Implements IMovieRepository

    Public Function ListMovies() As IEnumerable(Of Movie) Implements IMovieRepository.ListMovies
        Return Nothing
    End Function

    Public Sub CreateMovie(ByVal movieToCreate As Movie) Implements IMovieRepository.CreateMovie
    End Sub

End Class
